module/TaskManagement/test/integration/BaseTaskProcessTest.php

```diff
@@ -52,4 +61,42 @@ protected function setupController($controllerName, $actionName)
         $pluginManager = $this->serviceManager->get('ControllerPluginManager');
         $this->controller->setPluginManager($pluginManager);
     }
+
+    protected function generateRandomName() {
```

Comment:
valuterei se spostare questi metodi e quelli sotto in un oggetto che si occupa delle fixtures, per evitare di dare troppa responsabilità a questa classe